modules/sdk/project-templates/project-templates/src/test/java/com/liferay/project/templates/ProjectTemplatesTest.java

```diff
@@ -535,6 +535,49 @@ public void testBuildTemplateControlMenuEntryWithBOM() throws Exception {
 			gradleProjectDir, "build.gradle", _DEPENDENCY_PORTAL_KERNEL + "\n");
 	}
 
+	@Test
+	public void testBuildTemplateExt() throws Exception {
+		File gradleProjectDir = _buildTemplateWithGradle(
+			"modules-ext", "loginExt", "--original-module-name",
+			"com.liferay.login.web", "--original-module-version", "1.0.0");
+
+		_testContains(
+			gradleProjectDir, "build.gradle", "buildscript {", "repositories {",
+			"originalModule group: \"com.liferay\", name: " +
+				"\"com.liferay.login.web\", version: \"1.0.0\"",
+			"apply plugin: \"com.liferay.osgi.ext.plugin\"");
+
+		_executeGradle(gradleProjectDir, _GRADLE_TASK_PATH_BUILD);
+
+		_testExists(
+			gradleProjectDir, "build/libs/com.liferay.login.web-1.0.0.ext.jar");
+	}
+
+	@Test
+	public void testBuildTemplateExtInWorkspace() throws Exception {
+		File workspaceDir = _buildWorkspace();
+
+		File extDir = new File(workspaceDir, "ext");
```

Comment:
The `extDir` variable is used once, so it's better to use it inline on `L563` to improve readability.

```
File workspaceProjectDir = _buildTemplateWithGradle(
	new File(workspaceDir, "ext"), "modules-ext", "loginExt",
	"--original-module-name", "com.liferay.login.web",
	"--original-module-version", "1.0.0");
```